cli/tests/unit/version_test.ts

```diff
@@ -8,6 +8,7 @@ Deno.test(function version() {
   assert(
     pattern.test(Deno.version.typescript) ||
       Deno.version.typescript === "0-dev" ||
-      Deno.version.typescript === "0-beta",
+      Deno.version.typescript === "0-beta" ||
+      Deno.version.typescript === "1-rc",
```

Comment:
Why is this change required? Does it come from the time you were using one of `rc` versions of TS?